rasa_core/policies/ensemble.py

```diff
@@ -350,6 +351,21 @@ def probabilities_using_best_policy(self,
                 result = probabilities
                 best_policy_name = 'policy_{}_{}'.format(i, type(p).__name__)
                 best_policy_priority = p.priority
+                best_policy_index = i
+
+        if logger.level == logging.DEBUG:
```

Comment:
Not sure about our general coding practices here (perhaps some input from @tmbo would help) -- it makes sense to only go through this logic if the logger is on debug, but I don't see anywhere else in the code where take the logger level into account